app/helpers/startcase-string.js

```diff
@@ -0,0 +1,6 @@
+import Ember from "ember";
+import _ from "lodash";
+
+export default Ember.Helper.helper(function(value) {
+  return _.startCase(value[0]);
+});
```

Comment:
Isn't that the same as capitalize ?

Reply:
This removes any separator like  underscore, hyphen etc and make it capitalize.